recipe/meta.yaml

```diff
@@ -70,13 +70,14 @@ test:
     - pip check
 
 about:
-  home: http://flaskappbuilder.pythonanywhere.com/
+  home: https://flaskappbuilder.pythonanywhere.com/
   license_file: LICENSE
   license: BSD-3-Clause
   license_family: BSD
   summary: Simple and rapid application development framework, built on top of Flask. includes detailed security, auto CRUD generation for your models, google charts and much more.
+  description: Simple and rapid application development framework, built on top of Flask.
```

Comment:
The summary is longer than the description. It should be the opposite

Reply:
Okay, thanks!